examples/caa/caa.ipynb

```diff
@@ -0,0 +1,766 @@
+      "Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.111.133|:443... connected.\n",
+      "HTTP request sent, awaiting response... "
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
+      "To disable this warning, you can either:\n",
+      "\t- Avoid using `tokenizers` before the fork if possible\n",
+      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "200 OK\n",
+      "Length: 17267 (17K) [application/octet-stream]\n",
+      "Saving to: ‘vec_layer_15_Llama-2-7b-chat-hf.pt.1’\n",
+      "\n",
+      "vec_layer_15_Llama- 100%[===================>]  16.86K  --.-KB/s    in 0s      \n",
+      "\n",
+      "2024-01-28 11:45:52 (71.4 MB/s) - ‘vec_layer_15_Llama-2-7b-chat-hf.pt.1’ saved [17267/17267]\n",
+      "\n",
+      "--2024-01-28 11:45:52--  https://raw.githubusercontent.com/nrimsky/CAA/main/vectors/vec_layer_15_Llama-2-13b-chat-hf.pt\n",
+      "Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.110.133, 185.199.109.133, 185.199.108.133, ...\n",
+      "Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.110.133|:443... connected.\n",
+      "HTTP request sent, awaiting response... 200 OK\n",
+      "Length: 11126 (11K) [application/octet-stream]\n",
+      "Saving to: ‘vec_layer_15_Llama-2-13b-chat-hf.pt.1’\n",
+      "\n",
+      "vec_layer_15_Llama- 100%[===================>]  10.87K  --.-KB/s    in 0s      \n",
+      "\n",
+      "2024-01-28 11:45:53 (99.9 MB/s) - ‘vec_layer_15_Llama-2-13b-chat-hf.pt.1’ saved [11126/11126]\n",
+      "\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
+      "To disable this warning, you can either:\n",
+      "\t- Avoid using `tokenizers` before the fork if possible\n",
+      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Download the CAA sycophancy vectors for layer 15\n",
+    "!wget https://raw.githubusercontent.com/nrimsky/CAA/main/vectors/vec_layer_15_Llama-2-7b-chat-hf.pt\n",
+    "!wget https://raw.githubusercontent.com/nrimsky/CAA/main/vectors/vec_layer_15_Llama-2-13b-chat-hf.pt"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Cosine similarity: 0.995\n"
```

Comment:
🥇 This is awesome to see in the notebook! Really powerful validation this is doing the right thing.